credentials/src/test/kotlin/SSITest.kt

```diff
@@ -108,6 +111,40 @@ class SSITest {
     assertTrue(VerifiableCredential.verify(vcJwt, SimpleResolver(didDocument)))
   }
 
+  @Test
+  fun `creates credential status vc with valid vc builder returns vc jwt`() {
```

Comment:
Is credential status meant to be part of this PR?

Reply:
No it snuck in